tests/test_applets/blockwiseObjectClassification/testOpBlockwiseObjectClassification.py

```diff
@@ -48,7 +49,8 @@
 
 WRITE_DEBUG_IMAGES = False
 
-class TestOpBlockwiseObjectClassification(object):
+
+class TestOpBlockwiseObjectClassification(unittest.TestCase):
```

Comment:
There are two modes of discovery of `setUp` and `tearDown` methods. If class is inherited from `unittest.TestCase` then `setUp` and `tearDown` picked up by pytest otherwise they should be called `setup_method(self, method)` and `teardown_method(self, method)`. I chose less invasive approach where I can.

Reply:
cool